frontend/app/components/mapbox/current-mouse-position.js

```diff
@@ -13,4 +13,14 @@ export default class MapboxCurrentMousePositionComponent extends Component {
   handleMouseLeave() {
     this.set('currentMapMouseEvent', null);
   }
+
+  @computed('currentMapMouseEvent')
+  get mousePosition() {
```

Comment:
Is this also something that can/should be called in `didUpdateAttrs`?

Reply:
Oh my gosh yes, I can't believe I overlooked didUpdateAttrs. I maybe got too carried away/focused on decorators? 

These should definitely be in didUpdateAttrs.

Reply:
I think I follow - we've used `didUpdateAttrs` in other similar approaches before.

One thing this computed property does that made lead to unintended consequences is that it fires an action. This is a side effect of simply invoking `mousePosition` as a property somewhere else, which might be surprising. I think it's okay for now but I think using `didUpdateAttrs` is clearer.